cider-velox/src/ArrowConvertorUtils.cpp

```diff
@@ -22,6 +22,7 @@
 #include "ArrowConvertorUtils.h"
 #include "BitUtils.h"
 #include "TypeConversions.h"
+#include "cider/CiderException.h"
```

Comment:
still has CiderException.h in cider-velox folder

Reply:
fix